src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/FallbackRazorConfiguration.cs

```diff
@@ -34,6 +34,12 @@ internal class FallbackRazorConfiguration : RazorConfiguration
              "MVC-3.0",
              new[] { new FallbackRazorExtension("MVC-3.0"), });
 
+        public static readonly RazorConfiguration MVC_5_0 = new FallbackRazorConfiguration(
+             RazorLanguageVersion.Version_5_0,
+             // Razor 5.0 uses MVC 3.0 Razor configuration.
+             "MVC-3.0",
```

Comment:
Nit: private const for this string.

Reply:
This file uses non-const strings. I'll stick with the theme for now.